README.md

```diff
@@ -582,9 +582,24 @@ http://www.codetriage.com/potenza/bootstrap_form
 
 ## Contributing
 
+The release of Bootstrap 4 and Rails 5.1 have implications
+for the future direction of `bootstrap_form`.
+Don't worry.
+We plan to move this gem forward to Bootstrap 4
+and to support Rails 5.1 and beyond.
+If you're thinking of contributing to `bootstrap_form`,
+please read issue #361.
```

Comment:
Could you hyperlink the issue number? GitHub doesn't seem to do it automatically in this context.